brayns/common/utils/Utils.h

```diff
@@ -34,4 +36,18 @@ inline auto lowerCase(std::string str)
     std::transform(str.begin(), str.end(), str.begin(), ::tolower);
     return str;
 }
+
+template <typename T>
+inline T enumSet(const T eA, const T eB)
```

Comment:
```suggestion
template <typename T>
inline typename std::underlying_type<T>::type setBit(const std::underlying_type<T>::type mask, const T bit)
{
     return mask | static_cast<typename std::underlying_type<T>::type>(bit);
}
```

You should return the underlying type, because semantically the output value is not the enum anymore.
But even worse, imagine you have this code

    enum class A : int { a = 1, b = 2 };
    int f(const A a)
    {
        switch(a)
        {
            case A::a: return int(a);
            case A::b: return int(a);
        }
    }

That code if perfect, but if you call f with something A(17), the function return value will be undefined.